ML/config_Training_LctopK0s_13TeV_pt_2_4.yml

```diff
@@ -0,0 +1,168 @@
+input: # files to use, set FD to null for binary classification
+    prompt:
+        [
+            ~/DmesonAnalysis/filterdata/filtered/13TeV/Prompt_filter_pT_2_4.root,
+            #~/DmesonAnalysis/filterdata/filtered/13TeV/Prompt_filter_pT_4_6.root,
+            #~/DmesonAnalysis/filterdata/filtered/13TeV/Prompt_filter_pT_6_50.root,
+        ]
+    FD:
+        [
+            ~/DmesonAnalysis/filterdata/filtered/13TeV/FD_filter_pT_2_4.root,
+            #~/DmesonAnalysis/filterdata/filtered/13TeV/FD_filter_pT_4_6.root,
+            #~/DmesonAnalysis/filterdata/filtered/13TeV/FD_filter_pT_6_50.root,
+        ]
+
+    data:
+        [
+            ~/DmesonAnalysis/filterdata/filtered/13TeV/Data_filter_pT_2_4.root,
+            #~/DmesonAnalysis/filterdata/filtered/13TeV/Data_filter_pT_4_6.root,
+            #~/DmesonAnalysis/filterdata/filtered/13TeV/Data_filter_pT_6_50.root,
+        ]
+    treename: treeMLLc
+
+output:
+    leg_labels: ["Background", "Prompt L$_c$", "Feed-down L$_c$"] # legend labels, keep the right number of classes
+    out_labels: ["Bkg", "Prompt", "FD"] # output labels, keep the right number of classes
+    dir: ~/DmesonAnalysis/ML/trained_models/LctopK0s/13TeV/ # output dir
+
+pt_ranges: # ranges in pt to split the data in the ml training and testing
+    min: [2] # list
+    max: [4] # list
+
+data_prep:
+    filt_bkg_mass: inv_mass < 2.15 or inv_mass > 2.4 # pandas query to select bkg candidates
+    dataset_opt:
+        "max_signal" # change how the dataset is built, options available: 'equal', 'max_signal'
+        # 'equal' -> same number of prompt/FD/bkg (not using all the signal available)
+        # 'max_signal' -> try to use all the signal (prompt and FD) + add n_bkg = bkg_mult * (n_prompt + n_FD)
+    bkg_mult: [0.08] # list of multipliers for (nPrompt + nFD) used to determine nCandBkg in the 'max_signal' option
```

Comment:
why did you set an amount of background which is only 0.08 times the signal? Maybe it would be better to use the same amount of signal and background at low pT where we don't have problems with the statistics